src/pkgdev/scripts/pkgdev_mask.py

```diff
@@ -217,12 +223,13 @@ def _mask(options, out, err):
     p = git.run('config', 'user.email', stdout=subprocess.PIPE)
     email = p.stdout.strip()
 
+    bugs = options.bugs if options.bugs else []
```

Comment:
The default argparse value for the append action should return an empty list so this is unnecessary.

Reply:
When I just printed it, I got `None`, that's why I did it this way. But I'll check again.

Reply:
There's a way to make it default to the empty list instead of None then... I remember doing it. :smile:

Reply:
Maybe `default=[]`?

Reply:
I'll figure it out.

Reply:
`default=[]` it was :-)